spring-cloud-dataflow-rest-resource/src/main/java/org/springframework/cloud/dataflow/rest/util/DeploymentPropertiesUtils.java

```diff
@@ -149,7 +149,7 @@ private DeploymentPropertiesUtils() {
 	 */
 	public static List<String> parseArgumentList(String s, String delimiter) {
 		ArrayList<String> pairs = new ArrayList<>();
-		if (s != null && s.contains("=")) {
+		if (s != null) {
```

Comment:
Thank you for your contribution.   This solution causes a side effect to the current functionality. Here is a test with the expected results based on current behavior with your changes:
```
props = DeploymentPropertiesUtils.parseArgumentList("boo foo=bar xxx3", " ");
assertTrue(props.contains("boo"));
assertTrue(props.contains("foo=bar xxx3"));
```
But the result of the parse produces:
* boo
* foo=bar
* xxx3

Currently, the code looks at embedded blanks as part of the value.   With the solution provided this would use the spaces as a delimiter and split a value into multiple keys.